### ISSUE TITLE
Can't open new window on separate desktop (macOS) if there are windows on other desktops

### ISSUE THREAD

**Issue by chpmrc** [microsoft/vscode#63959]
- VSCode Version: 1.29.1 and 1.30.0-insider
- OS Version: macOS 10.14 (18A391)

Steps to Reproduce:

1. Open VSCode in Desktop 1
2. Move to Desktop 2
3. "New Window (Cmd+Shift+N)" won't work
4. Right click the icon in the Dock -> "New Window" will instead work

Gif showing the bug: https://duaw26jehqd4r.cloudfront.net/items/1V0c0V0k3w3d313a2b1m/Screen%20Recording%202018-11-29%20at%2001.29%20AM.gif

Does this issue occur when all extensions are disabled?: **Yes**

**Comment by chpmrc** [comment]
The problem described in #35240 is similar but not the same (in that case it focuses on the existing virtual desktop).

**Comment by jlarcombe** [comment]
I wonder if this is the same thing as something I've been seeing:

1. Run VS.code.
2. Create a window.
3. Minimise to the dock (with the yellow traffic-light icon in the window).
4. Press Shift-Cmd-N to create another new window.

EXPECTED: The new window appears.
ACTUAL: Nothing happens.

**Comment by chpmrc** [comment]
Yep, can confirm that happens too. Probably related. Thanks!

**Comment by chpmrc** [comment]
Can we remove the `insiders` label? This happens with the stable release too. Thanks.

**Comment by chpmrc** [comment]
As a workaround I created an Automator application called "VSCode New Window" that runs a "Shell Script" (one of the available actions in Automator) with the command `/usr/bin/local/code -n`.

**Comment by kkew3** [comment]
Still an issue on VSCode 1.34.0 / macOS 10.14.4.

Besides "New File", it cannot "Open...", "Open Workspace...", etc. "Open Recent..." opens a window but it will jump to another space, which is not desirable.

**Comment by SamWolski** [comment]
Still an issue on VSCode 1.35.1 / macOS 10.14.5

**Comment by bzmillerboy** [comment]
Still an issue on VSCode 1.42.0 / macOS 10.15.3

**Comment by robderon** [comment]
Still an issue on VSCode 1.43.2 / macOS 10.15

**Comment by rlab** [comment]
Still an issue on VSCode 1.44 / macOS 10.15.4

**Comment by wmayner** [comment]
Still an issue on 
```
Version: 1.47.0-insider
Commit: 9bf6af04dcb5c2e531cedca83cd188939ca1b0db
Date: 2020-06-26T21:11:02.609Z
Electron: 8.3.3
Chrome: 80.0.3987.165
Node.js: 12.13.0
V8: 8.0.426.27-electron.0
OS: Darwin x64 19.5.0
```

**Comment by squarism** [comment]
Why was this closed?  I think people are commenting and replying to this issue (for years) to explicitly let the team that this hasn't been fixed.  Or possibly people have been replying so it doesn't get lost or be assumed to be fixed with Mac updates, library updates or other passive luck.

Is it a `wontfix` or just too mysterious, related to Electron or some other fundamental technology that it has become `cantfix`?

**Comment by robenkleene** [comment]
I think it's been closed because it's been fixed. This commit was merged to master early today https://github.com/microsoft/vscode/commit/6dc0327c9af20d0dfe7112feed7c4f75d32b0e83 🎉🎉🙏 Thank you team!

**Comment by sbatten** [comment]
The unreleased tag should be updated when the fix ships in insiders in the coming days.

**Comment by squarism** [comment]
> I think it's been closed because it's been fixed. This commit was merged to master early today [6dc0327](https://github.com/microsoft/vscode/commit/6dc0327c9af20d0dfe7112feed7c4f75d32b0e83) 🎉🎉🙏 Thank you team!

Oh, I see!  I missed the SHA/commit.  Sorry.  🙂
